### ISSUE TITLE
Use a rotation mode instead of having separate top and log blocks for wood and cacti.

### ISSUE THREAD

**Issue by IntegratedQuantum** [PixelGuys/Cubyz#357]
![abc](https://github.com/PixelGuys/Cubyz/assets/43880493/f469f3c3-15b9-41a0-b86d-cda44dd21e47)

**Comment by archbirdplus** [comment]
Do you intend to gracefully handle multiple block thick trunks with log rotation?

**Comment by IntegratedQuantum** [comment]
No, in my opinion that would be too many things at once. And you'd also need textures for multi block logs too.
I think that should be handled by a separate block, or is it even worth handling at all? e.g. thaumcraft's large trees also just use normal logs and no one cares.

**Comment by du82** [comment]
I have issue with this because I want to build with the ends of the logs as their own texture, and directionally as well.
